frontend/src/metabase/app.js

```diff
@@ -83,6 +84,7 @@ function _init(reducers, getRoutes, callback) {
   registerVisualizations();
 
   store.dispatch(refreshSiteSettings());
+  store.dispatch(setIsDndAvailable(true));
```

Comment:
This is only run when the original app is initialized, ensuring that we can still use the Collection browser when we don't have the drag and drop provider

Reply:
Yeah I can set the initial state of the SDK store to `isDndAvailable: false` and that should do the trick.